core/templates/dev/head/domain/topic/SubtopicObjectFactorySpec.ts

```diff
@@ -16,13 +16,20 @@
  * @fileoverview Tests for SubtopicObjectFactory.
  */
 
+import { SkillSummaryObjectFactory } from
```

Comment:
Do we also have to remove this after angular 8 upgraded, if yes then let's add a TODO comment.

Reply:
Done!